### ISSUE TITLE
Clearer structures

### ISSUE THREAD

**Issue by Pfeifakopf** [statamic/ideas#103]
If you have a complex structure tree with lot of entries and a depth of 3+, it's really hard to orientate and see, on which levels the entries are. Perhaps you could differentiate the levels by colors or lines better.
![image](https://user-images.githubusercontent.com/3351449/77157892-8fead700-6aa2-11ea-9fc0-8a53637754d4.png)

**Comment by jeremydouglas** [comment]
I agree, I would be nice even to have a custom field that could be shown in this view. Maybe even a thumbnail or icon.

Making the parents collapsable would also help.
